### DISCUSSION TITLE
how to increase time for super key?

### DISCUSSION THREAD

**Discussion by vekexasia** [manna-harbour/miryoku#326] [Q&A] [answered]
i m learning miryoku and my brain still farts so sometimes it takes a bit for me to type the 'a' letter which is the super key in the qwerty layout

**Comment by vekexasia** [comment]


**Reply by vekexasia** [reply]
closing as I got used to it

**Comment by manna-harbour** [comment, marked as answer]
Speeds vary between fingers, so I plan on adding per-key tapping term (https://github.com/manna-harbour/miryoku/issues/33) for each finger, with pinkie being the slowest.

You could just increase the tapping term for all fingers (https://github.com/manna-harbour/miryoku_zmk/blob/e6683e9f8b6c199b339208b1b501e88a7308ed48/miryoku/miryoku.h#L26), but to try per-key tapping term, you'd add new mod tap behaviors for each finger to https://github.com/manna-harbour/miryoku_zmk/blob/master/miryoku/miryoku_behaviors.dtsi, and substitute a custom Base layer using the behaviors with https://github.com/manna-harbour/miryoku/discussions/85.
